config.cfg.example

```diff
@@ -3,6 +3,6 @@ DISCORD_TOKEN = "ADD_TOKEN"
 BACKUP_TIME = 6
 # Number of backups to keep
 KEEP_BACKUPS = 5
-COMMANDS_LINK = "https://link-to-commands"
+COMMANDS_LINK = "https://rayneh.github.io/Pubobot/commands.md"
```

Comment:
idk if we wanna do this on my github or yours? i don't care where it's hosted, as long as we can be sure it's up to date

Reply:
should be on yours, check the TODO list. ill try mkdocs today tho before we merge this. At the moment its just displaying the pure .md file